1.process_data/nbconverted/process_jump_pilot.py

```diff
@@ -210,7 +233,16 @@ def fill_dmso(df):
         meta_features="infer",
         samples="Metadata_control_type == 'negcon'",
         method="standardize",
-        output_file=output_file,
+        output_file=normalized_output,
+        output_type="parquet",
+    )
+
+    # Feature select normalized data
+    feature_select(
```

Comment:
Everything looks good here! Note that the correlation threshold takes the longest to run, so it might take a while for this to run for all 51 plates. Nice job!

Reply:
Good to know, thanks!